mne/source_estimate.py

```diff
@@ -2115,8 +2131,10 @@ class MixedSourceEstimate(_BaseSourceEstimate):
         a tuple with two arrays: "kernel" shape (n_vertices, n_sensors) and
         "sens_data" shape (n_sensors, n_times). In this case, the source
         space data corresponds to "numpy.dot(kernel, sens_data)".
-    vertices : list of array
-        Vertex numbers corresponding to the data.
+    vertices : list of array, shape (2,)
+        Vertex numbers corresponding to the data. The first element of the list
+        contains vertices of left hemisphere and the second element contains
+        vertices of right hemisphere.
```

Comment:
I don't think this is correct for a MixedSourceEstimate

Reply:
hmm ... but see this: https://github.com/mne-tools/mne-python/blob/master/mne/source_estimate.py#L2160